src/constants.ts

```diff
@@ -0,0 +1,5 @@
+export enum DecoratorKeys {
```

Comment:
Could you move this into `src/internal`?

Reply:
Sure I will move the file

Reply:
when this is moved, i could merged these changes, then you could do https://github.com/hasezoey/typegoose/pull/18#issuecomment-527388539, thanks

PS: could you rebase & squash your changes?

Reply:
Made it, i will now Squash & Merge it